src/resources/js/views/month-mobile-events.js

```diff
@@ -225,11 +275,19 @@ tribe.events.views.monthMobileEvents = {};
 		var containerState = $container.data( 'tribeEventsState' );
 		var isMobile = containerState.isMobile;
 
-		if ( ! isMobile && ! state.desktopInitialized ) {
-			obj.closeAllEvents( $container );
-			state.desktopInitialized = true;
-		} else if ( isMobile && state.desktopInitialized ) {
-			state.desktopInitialized = false;
+		if ( ! isMobile ) {
+			$( obj.selectors.mobileEventsDefaultNotices ).hide();
+			$( obj.selectors.mobileEventsDynamicNotices ).hide();
+
+			if ( ! state.desktopInitialized ) {
+				obj.closeAllEvents( $container );
+				state.desktopInitialized = true;
+			}
+		} else {
+			obj.handleMobileDayClick();
+			if ( state.desktopInitialized ) {
+				state.desktopInitialized = false;
+			}
 		}
```

Comment:
Maybe a personal preference but you can get ride of the `! isMobile `

```suggestion
		if ( isMobile ) {
			obj.handleMobileDayClick();
			if ( state.desktopInitialized ) {
				state.desktopInitialized = false;
			}
		} else {
			$( obj.selectors.mobileEventsDefaultNotices ).hide();
			$( obj.selectors.mobileEventsDynamicNotices ).hide();

			if ( ! state.desktopInitialized ) {
				obj.closeAllEvents( $container );
				state.desktopInitialized = true;
			}
		}
```